.github/workflows/python-app-macos-windows.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ macos-latest, windows-latest ]
+        os: [ macos-13, windows-latest ] # Using macos-13 since macos-latest no longer supports 3.8
```

Comment:
Python is on 3.13 now, isn't it? Something to tackle another time 😭

Reply:
Yeah 3.13 is the latest version! I thought about maybe trying to finally update the Python version but figured that deserves its own ticket (though I don't think the migration will be all that difficult)